db/migrate/20160901010430_change_column_type.rb

```diff
@@ -1,13 +0,0 @@
-class ChangeColumnType < ActiveRecord::Migration
```

Comment:
![screenshot_1](https://cloud.githubusercontent.com/assets/13460052/18411112/e0298e6a-7725-11e6-8e0a-2b32b316c16b.png)
I could be wrong, but I think they are both using a float.  I was getting an error because both file names are the same, "change_column_type.rb".  Should I try to rename one of them to keep both?